src/ripple/rpc/handlers/AccountTx.cpp

```diff
@@ -236,7 +236,7 @@ getLedgerRange(
                         uLedgerMax = ls.max;
                     }
                     if (uLedgerMax < uLedgerMin)
-                        return rpcLGR_IDXS_INVALID;
```

Comment:
Unless I am mistaken, if the user passes invalid ledger indices, where the min is greater than the max, the error returned will be `rpcNOT_SYNCED`. But that's not an accurate description of the error, because the server could actually be synced.

Reply:
Yes, let's not return `rpcNOT_SYNCED` in the case where the server actually _is_ synced but the user submits invalid parameters.

`lgrIdxsInvalid` (the code that actually gets returned in the API for `rpcLGR_IDXS_INVALID`) is very hard to read, though, and redundant with other errors as well.

`rpcINVALID_LGR_RANGE` is a sensible code also used in some other places that would work instead for the case where the user supplied an invalid range. Or you could use [`invalidParams`](https://github.com/ripple/rippled/search?p=2&q=invalidParams&unscoped_q=invalidParams), which is returned in some cases as well.

Reply:
I've changed this to `rpcINVALID_LGR_RANGE`.